### DISCUSSION TITLE
Sparrow VM Sizing

### DISCUSSION THREAD

**Discussion by Bharath-TSI** [caas-team/sparrow#157] [Q&A] [answered]
Do we have any recommended VM sizing requirement documentation?

**Comment by niklastreml** [comment, marked as answer]
We don't have any recommendations currently, as the actual resource usage depends on the config you use. As far as our internal deployments go, we're seeing ~5 mCPU and ~35 MiB memory usage on average for the sparrow container with 4 to 5 targets per check. So you really shouldn't need a beefy machine to run it
